package/src/extra/db.d.ts

```diff
@@ -0,0 +1,8 @@
+type DbHooks = Omit<
+	import("@astrojs/db/types").AstroDbIntegration["hooks"],
+	keyof import("astro").AstroIntegration["hooks"]
+>;
+
+declare namespace AIK {
```

Comment:
Bit of a nitpick, but how do we feel about this being namespace` AIK` and not `AstroIntegrationKit` ?